src/pip/_internal/cli/progress_bars.py

```diff
@@ -61,7 +65,8 @@ def _raw_progress_bar(
     iterable: Iterable[bytes],
     *,
     size: Optional[int],
-) -> Generator[bytes, None, None]:
+    unit_size: Optional[int],
+) -> Generator[Any, None, None]:
```

Comment:
Should the type in `Iterable` and `Generator` match? This should probably use a TypeVar if so.